src/components/text-editor/text-editor.component.js

```diff
@@ -65,6 +65,7 @@ const TextEditor = React.forwardRef(
     const [inlines, setInlines] = useState([]);
     const [activeInlines, setActiveInlines] = useState({});
     const [focusToolbar, setFocusToolbar] = useState(false);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I've pulled the branch down locally and this doesn't flag, you should remove this